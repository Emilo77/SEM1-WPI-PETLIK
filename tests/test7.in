ggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
(ga(gbb(gccc(gdddd(geeee(gfffff))))))
=a
=b
=c
=d
=e
=f
=g
