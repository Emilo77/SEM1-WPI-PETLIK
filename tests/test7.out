1
2
3
4
4
690
0
